independent-publisher-connectors/SureXeroLite/README.md

```diff
@@ -0,0 +1,26 @@
+# SureXeroLite (The 848 Group)
+

```

Comment:
Kindly refer the below link and update the readme.md file, mainly please add the required fields.

Reply:
Thanks for you comment. I don't think I understand that. Where is the link? and what fields I need to add? Thanks Vadim

Reply:
It's my bad forgot to paste the link. 
here you go. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Reply:
I have updated the readme.md file/